src/classes.scss

```diff
@@ -39,3 +39,4 @@
 @import './tree/classes';
 @import './user-icon/classes';
 @import './wizard/classes';
+@import './carousel/classes';
```

Comment:
This needs to be alphabetized. Same in index.scss

Reply:
Alphabetized both classes and index in the next push.